scripts/get-nteract.sh

```diff
@@ -0,0 +1,10 @@
+sudo apt install libgif-dev
+git clone git@github.com:nteract/nteract.git
+cd nteract
+yarn
+cd applications/desktop
+yarn dist
+cd dist
+AppImage=$(ls nteract*.AppImage)
```

Comment:
Awesome. Might be worth making this something like `ls -l nteract*.AppImage | head -n 1` and also checking the return code. This is forward progress though so let's merge it!